R/util.R

```diff
@@ -21,6 +21,14 @@ read_json <- function(path) {
 }
 
 
+read_string_as_json <- function(path) {
+  str <- paste(readLines(path), collapse = "")
```

Comment:
I think this needs to be `collapse = "\n"`

Reply:
In fact it needs to not be `collapse = "\n"`  because `\n` is not valid in a json string. And no need for line breaks in the response because it's going to be rendered as html...

Reply:
Right, of course on the newlines in the json.

But, if you collapse with "" though you remove whitespace which could change the interpretation. If it can't be "\n" should it be " "?

Reply:
that might be a safer bet yeah